spirv_glsl.cpp

```diff
@@ -48,6 +48,36 @@ enum ExtraSubExpressionType
 	EXTRA_SUB_EXPRESSION_TYPE_AUX = 0x20000000
 };
 
+const std::vector<std::tuple<uint32_t, const char *>> COOPVEC_COMPONENT_TYPE_NAMES = {
```

Comment:
This should just be a static const array of POD struct inside the function that uses it.
Having a global const of non-POD types requires global initialization which really should be avoided if we can help it.